<ui version="4.0" >
 <class>OrbitalDialog</class>
 <widget class="QDialog" name="OrbitalDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Orbitals</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLineEdit" name="fileName" />
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="loadFile" >
     <property name="text" >
      <string>Load File</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="calculate" >
     <property name="text" >
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>177</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QComboBox" name="orbitalCombo" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OrbitalDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OrbitalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
